src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,23 @@
+import { constants } from "../utils/ApiConstants";
+import { getKeyFromSession, startStubServiceAndReturnSessionIdByType } from "../utils/ApiTestSteps";
+
+
+describe("/session Happy Path", () => {
+	it("BAV | NO_PHOTO_ID test", async () => {
+		const sessionResponse = await startStubServiceAndReturnSessionIdByType("FACE_TO_FACE");
+		expect(sessionResponse.status).toBe(200);
+		console.log(sessionResponse.data);
+		const sessionId = sessionResponse.data.session_id;
+		console.log(sessionId);
+		await expect(getKeyFromSession(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey")).resolves.toBe("FACE_TO_FACE");
+	});
+
+	it("F2F | FACE_TO_FACE test", async () => {
```

Comment:
These two tests look the same apart from the values, I reckon we can use an [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) what do you think?